storybook/web/preview.js

```diff
@@ -2,8 +2,7 @@ import React from 'react';
 import { ThemeProvider } from 'styled-components';
 import { addDecorator } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs/react';
-
-const theme = {};
+import theme from '../../src/tokens/theme';
```

Comment:
This will restrict us to only light theme. We need ThemeSwitcher. Maybe not in this PR but we should take it up quickly

Reply:
Sure